js/src/collapse.js

```diff
@@ -224,8 +233,6 @@ class Collapse {
 
     this._element.style[dimension] = `${this._element.getBoundingClientRect()[dimension]}px`
 
-    Util.reflow(this._element)
```

Comment:
Maybe this is the cause for the collapse issue?

Reply:
No it was a little trick to force IE to refresh the UI